pystiche_papers/ulyanov_et_al_2016/utils.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, List, Optional
+
+from torch import nn, optim
+from torch.optim.lr_scheduler import ExponentialLR
+from torch.optim.optimizer import Optimizer
+
+from pystiche.enc import MultiLayerEncoder, vgg19_multi_layer_encoder
+from pystiche.image.transforms.processing import CaffePostprocessing, CaffePreprocessing
+
+
+def ulyanov_et_al_2016_multi_layer_encoder() -> MultiLayerEncoder:
+    return vgg19_multi_layer_encoder(
+        weights="caffe", internal_preprocessing=False, allow_inplace=True
+    )
+
+
+def ulyanov_et_al_2016_preprocessor() -> CaffePreprocessing:
+    return CaffePreprocessing()
+
+
+def ulyanov_et_al_2016_postprocessor() -> CaffePostprocessing:
+    return CaffePostprocessing()
+
+
+def ulyanov_et_al_2016_optimizer(
+    transformer: nn.Module, impl_params: bool = True, instance_norm: bool = True
+) -> Optimizer:
```

Comment:
Make this more specific
```suggestion
) -> optim.Adam:
```